caffe2/operators/resize_op.cc

```diff
@@ -151,19 +168,29 @@ output_width = floor(input_width * width_scale)
 output_height = floor(output_height * height_scale)
 )DOC")
     .Input(0, "X", "Input tensor")
+    .Input(1, "scales", // the hack to support onnx spec
+        "4D, Scales tensor, [1., 1., width_scale, height_scale]")
```

Comment:
why do we need to store the first two `1.`?

Reply:
I reserved it for the future... You are right, don't need that, let's make it aligned with argument version.